Samples/ResourceManagement/cpp-console-unpackaged/console_unpackaged_app.cpp

```diff
@@ -59,6 +60,13 @@ int wmain(int argc, wchar_t* argv[])
         return 1;
     }
 
+    // Despite the name of this API, it will return FALSE if the user is an admin but the context isn't elevated.
+    if (IsUserAnAdmin())
```

Comment:
I don't recommend doing this. Line 24 calls the bootstrapper. Today that does a FAIL_FAST_IF(is_elevated) so you'll never get here.

I'm changing MddBootstrapInitialize to return `HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED)` instead of FailFast so you could trap for that error explicitly, or just do nothing and let the current error handling handle it.

I recommend that rather than doing any explicitly elevation check like you're doing here